src/style.css

```diff
@@ -1 +1,32 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.stars__star {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
Last star should not have `margin-right`

Reply:
> Last star should not have `margin-right`

only last children in .stars--5?